src/backtrace/dbghelp.rs

```diff
@@ -162,7 +162,7 @@ pub unsafe fn trace(cb: &mut dyn FnMut(&super::Frame) -> bool) {
             context.ip(),
             fn_entry,
             &mut context.0,
-            &mut handler_data as *mut usize as *mut PVOID,
+            core::ptr::addr_of_mut!(handler_data) as *mut PVOID,
```

Comment:
Would it make sense to do something like `cast::<PVOID>()`?

Reply:
It would! And I would absolutely love to get rid of as many `as` casts throughout the code as possible (because it is essentially like C-style casting in C++: does too many different things at the same time), I just was not sure how welcome such changes would be.

Reply:
Very welcome, if you wouldn't mind!